app/models/user.rb

```diff
@@ -55,4 +55,21 @@ def create_token
       end while self.class.exists?(registration_token: registration_token)
     end
   end
+
+  def self.newtask(project)
```

Comment:
@pg20 why are they scopes? we want to find tasks for specific user so normal method would be sufficient